internal/conversionwebhook/resource_reservation.go

```diff
@@ -41,25 +39,19 @@ func init() {
 	_ = sparkscheme.AddToScheme(scheme)
 }
 
-// InitializeCRDConversionWebhook initialized conversion webhook and returns webhook client
-// configuration pointing to the webhook.
-func InitializeCRDConversionWebhook(
+// GetWebhookClientConfig returns webhook client configuration pointing to the webhook.
+func GetWebhookClientConfig(
 	ctx context.Context,
-	router wrouter.Router,
 	server config.Server,
 	schedulerNamespace string,
 	schedulerServiceName string,
 	schedulerServicePort int32,
+	schedulerServiceContextPath string,
 ) (*v1.WebhookClientConfig, error) {
-	err := addConversionWebhookRoute(ctx, router)
-	if err != nil {
-		return nil, err
-	}
-
-	path := filepath.Join(server.ContextPath, webhookPath)
+	path := filepath.Join(schedulerServiceContextPath, webhookPath)
```

Comment:
If the services are now totally separate, why do we need to merge the paths? Can we do this at the service level?

Reply:
We can have this all as config without joining. However, I'm not sure if we can set this at the service level, I think we'd need an ingress